src/main/java/io/confluent/connect/elasticsearch/ElasticsearchClient.java

```diff
@@ -554,9 +558,13 @@ private synchronized void reportBadRecord(BulkItemResponse response, long execut
           ? sinkRecords.get(response.getItemId())
           : null;
       if (original != null) {
+        // log only status, index and document id for record failure
         reporter.report(
             original,
-            new ReportingException("Indexing failed: " + response.getFailureMessage())
+            new ReportingException(
```

Comment:
Is there a possibility of `response.getFailure()` being null?
If yes, then better to check for null before accessing its status/index/id..

Reply:
This is being handled [here](https://github.com/confluentinc/kafka-connect-elasticsearch/blob/2604b9929bfe8c743b5dde2ae2a92330738ecea5/src/main/java/io/confluent/connect/elasticsearch/ElasticsearchClient.java#L562) & [here](https://github.com/confluentinc/kafka-connect-elasticsearch/blob/2604b9929bfe8c743b5dde2ae2a92330738ecea5/src/main/java/io/confluent/connect/elasticsearch/ElasticsearchClient.java#L570).